src/components/Main.js

```diff
@@ -5,8 +5,7 @@ class Main extends Component {
     render() {
         return (
             <div>
-               <HornedBeasts title="UniWhal" imageURL="http://3.bp.blogspot.com/_DBYF1AdFaHw/TE-f0cDQ24I/AAAAAAAACZg/l-FdTZ6M7z8/s1600/Unicorn_and_Narwhal_by_dinglehopper.jpg" description="A unicorn and a narwhal nuzzling their horns"/> 
-               <HornedBeasts title="Rhino Family" imageURL="https://images.unsplash.com/photo-1512636618879-bbe79107e9e3?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=bd9460ee6d1ddbb6b1ca7be86dfc4590&auto=format&fit=crop&w=1825&q=80" description="Parent rhino with two babies"/> 
+               let beastBoy = this.props.beastAll.map(beast => <HornedBeast newModalBeast={this.props.newModalBeast} beast={beast}/>)
             </div>
         );
     }
```

Comment:
HI Chris, good work so far on lab 2!

I cloned the repo down but errors are thrown:

` Line 8:65:   'HornedBeast' is not defined  react/jsx-no-undef
  Line 8:125:  'beast' is not defined        no-undef
`
- Check your import statement, you have horned beast imported as `HornedBeasts` but in the map you spelled the component `HornedBeast`. 
- Remove `beastBoy` variable. Instead when we map, we will want to use curlies first like so:
```
{
    data.map(beast => {
       return (<HornedBeast  />);
    })
`
- On the HornedBeast component in the map, we should use props for title, `{beast.title}`,  imageUrl, `{beast.image_url}` , and description , `{beast.description}`